src/pykeen/nn/representation.py

```diff
@@ -33,8 +33,9 @@
 from .combination import Combination, combination_resolver
 from .compositions import CompositionModule, composition_resolver
 from .init import initializer_resolver, uniform_norm_p1_
-from .text import TextEncoder, text_encoder_resolver
-from .utils import PyOBOCache, ShapeError, TextCache, WikidataCache
+from .text.text_cache import PyOBOTextCache, TextCache, WikidataTextCache
```

Comment:
Rename to `cache` and `encoder` since they are already in the `text` module?
```suggestion
from .text.cache import PyOBOTextCache, TextCache, WikidataTextCache
```

Reply:
Yes

Reply:
6c4beda1